### PR TITLE
Feature: historical non-article metrics

### PR DESCRIPTION
**Pull request by lsh-0**
adds support for changing GA query generation and results processing over time on per-pagetype.

### CODE REVIEW

**Comment by lsh-0** [comment]
opening up to review but I still need to:
* improve redirect file generation script.
* double check the data being scraped for the later page-types, they were added fairly quickly

**Comment by lsh-0** [comment]
I'd like to get this merged next week, so any feedback would be welcome.

File: src/metrics/interview_type.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from article_metrics.utils import lmap
+from .logic import generic_ga_filter
+
+def query_processor_frame_1(ptype, frame):
+
+    adhoc = lmap(lambda path: "ga:pagePath==%s" % path, [
+        "/from-ancient-dna-to-decay-an-interview-with-jessica-metcalf",
```
Comment (seanwiseman) [inline src/metrics/interview_type.py]: Could perhaps store these path values outside of the code? I see the ease of containing them here but might be easier to manage when the news items scale up in number.

Comment (lsh-0) [reply src/metrics/interview_type.py]: these are old urls, they're not going to change and we won't be supporting new news paths at old locations, if that makes sense ?

there are cases where there is a tremendous amount of redirects, and I have a simple parser happening for those, but for the most part they are contained within the code.

Comment (seanwiseman) [reply src/metrics/interview_type.py]: Ok, thank you for the context, that makes sense then.

**Comment by seanwiseman** [comment]
I have never used the `schema` library, only ever `json-schema` or something custom (which never seems to scale). Though looking through its repo and how you have applied it, it looks great. I will certainly try using it when I next need to.

**Pull request merged**